<launch>
	<group ns="einstein">
    <rosparam file="$(find robots_config)/einstein/config.yaml" command="load"/>
    <node name="pololu" pkg="ros_pololu_servo" type="ros_pololu_servo_node" output="screen">
          <param name="port" value="/dev/ttyACM0" />
    </node>
    <node name="pau2motors" pkg="pau2motors" type="pau2motors_node.py"
          output="screen"/>
	</group>
</launch>